phac-nml/irida
# Fix analysis results error loading

## Description of changes
What did you change in this pull request?  Provide a description of files changed, user interactions changed, etc.  Include how to test your changes.

Fixed UI loading issue on the analysis results page with an errored analysis temporarily displaying in the running state before showing the error state.

**To test**: 
Open up an errored analysis and the page should wait for the loading to complete before displaying to the user.

If you would like to see what issue this fixes you can switch to the development branch and open the errored analysis. You will see it flash the screen from a running state (while loading is happening) to the actual state of errored.

## Related issue
Link to the GitHub issue this pull request addresses using the `#issuenum` format.  If it completes an issue, use `Fixes #issuenum` to automatically close the issue.

## Checklist
Things for the developer to confirm they've done before the PR should be accepted:

~* [ ] CHANGELOG.md (and UPGRADING.md if necessary) updated with information for new change.~
~* [ ] Tests added (or description of how to test) for any new features.~
~* [ ] User documentation updated for UI or technical changes.~
